### DISCUSSION TITLE
Only view

### DISCUSSION THREAD

**Discussion by ChrisDorna** [ajaxorg/ace#5360] [General]
Is it possible to configure ACE for only viewing a file (including search, collapse etc.)

**Comment by andredcoliveira** [comment]
Hi, @ChrisDorna! The Ace editor instance has a method to set read-only mode: https://github.com/ajaxorg/ace/blob/c8379beca8988df59eb766f447a9ad461e6bc2d0/ace.d.ts#L926

You can find this and some other options in https://ace.c9.io/#nav=howto, under the **Common Operations** header.

**Reply by ChrisDorna** [reply]
Hello André,
Thanks for your answer, this setting did the job!
Chris

**Reply by andredcoliveira** [reply]
Glad to have helped!
